### PR TITLE
refact: remove source from kneighbor/kout results & unify 'degree ' error info

### PR DESCRIPTION
**Pull request by imbajin**
also adapt test in client

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/KoutTraverser.java [hunk 1]
```diff
@@ -121,9 +121,7 @@ public KoutTraverser(HugeGraph graph) {
         Set<Node> all = newSet(single);
 
         Node sourceV = new KNode(source, null);
-
         latest.add(sourceV);
```
[the remark this answers is not in the document]
Comment (imbajin) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/KoutTraverser.java]: won't return sourceV in any cases now

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/HugeTraverser.java [hunk 2]
```diff
@@ -141,16 +141,18 @@ protected int concurrentDepth() {
...
+                                         long remaining) {
         Set<Node> neighbors = newSet();
         for (Node source : vertices) {
             Iterator<Edge> edges = this.edgesOfVertex(source.id(), step);
             while (edges.hasNext()) {
                 Id target = ((HugeEdge) edges.next()).id().otherVertexId();
                 KNode kNode = new KNode(target, (KNode) source);
-                if ((excluded != null && excluded.contains(kNode)) ||
-                    neighbors.contains(kNode)) {
+                boolean exist = (excluded != null && excluded.contains(kNode));
```
Comment (zhoney) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/HugeTraverser.java:153]: rename `exist` to `matchExcluded` or `inExcluded`

**Review by zhoney — CHANGES_REQUESTED** [review]

**Review by zhoney — APPROVED** [review]

**Review by Linary — APPROVED** [review]

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/PersonalRankAPI.java [hunk 3]
```diff
@@ -69,8 +69,8 @@ public String personalRank(@Context GraphManager manager,
                         "The alpha of rank request must be in range (0, 1], " +
                         "but got '%s'", request.alpha);
         E.checkArgument(request.maxDegree > 0L || request.maxDegree == NO_LIMIT,
-                        "The max_degree of rank request must be > 0, " +
-                        "but got: %s", request.maxDegree);
+                        "The max_degree of rank request must be > 0 " +
+                        "or == -1, but got: %s", request.maxDegree);
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/PersonalRankAPI.java:73]: don't need "or == -1", expected all the checks keep the same style

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/HugeTraverser.java [hunk 4]
```diff
@@ -367,7 +370,7 @@ public static void checkSkipDegree(long skipDegree, long degree,
                         Query.DEFAULT_CAPACITY, skipDegree);
         if (capacity != NO_LIMIT) {
             E.checkArgument(degree != NO_LIMIT && degree < capacity,
-                            "The degree must be < capacity");
+                            "The max degree must be < capacity");
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/HugeTraverser.java:373]: max_degree

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/NeighborRankTraverser.java [hunk 5]
```diff
@@ -282,7 +282,8 @@ public void add(Node node) {
 
         public long degree() {
             E.checkArgument(this.degree > 0,
-                            "The degree must be > 0, but got %s", this.degree);
+                            "The max degree must be > 0, but got %s",
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/NeighborRankTraverser.java:285]: max_degree

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/steps/EdgeStep.java [hunk 6]
```diff
@@ -73,7 +73,7 @@ public EdgeStep(HugeGraph g, Directions direction, List<String> labels,
                     Map<String, Object> properties,
                     long degree, long skipDegree) {
         E.checkArgument(degree == NO_LIMIT || degree > 0L,
-                        "The degree must be > 0 or == -1, but got: %s",
+                        "The max degree must be > 0 or == -1, but got: %s",
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/steps/EdgeStep.java:76]: max_degree

**Review by javeme — CHANGES_REQUESTED** [review]

**Review by zhoney — DISMISSED** [review]

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/CountAPI.java [hunk 7]
```diff
@@ -75,7 +75,8 @@ public String post(@Context GraphManager manager,
                                "The steps of request can't be null or empty");
         E.checkArgumentNotNull(request.dedupSize == NO_LIMIT ||
                                request.dedupSize >= 0L,
-                               "The dedupSize of request must >= 0, but got '%s'",
+                               "The dedupSize of request " +
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/CountAPI.java:78]: dedup size

**Review by zhoney — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**